src/app/@ansyn/ol/maps/open-layers-map/openlayers-map/openlayers-map.ts

```diff
@@ -304,7 +304,9 @@ export class OpenLayersMap extends BaseImageryMap<OLMap> {
 	setMainLayerToBackgroundMap(layer: Layer) {
 		layer.set(ImageryLayerProperties.NAME, IMAGERY_MAIN_LAYER_NAME);
 		this.backgroundMapObject.getLayers().clear();
-		this.backgroundMapObject.addLayer(layer);
+		// TODO: Do we need to deep-copy the layer? Object.create does not do it...
+		const clonedLayer = Object.create(layer);
```

Comment:
why do you need to clone or create it?

Reply:
In OL6 we can't add the same layer to 2 different maps...